src/pages/developing/web-components-tutorial/overview.mdx

```diff
@@ -76,11 +76,11 @@ any step and take it from there.
 
 1. [**Installing Carbon**](/developing/web-components-tutorial/step-1)
```

Comment:
Ignacio just updated this section too in this [pr](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/pull/1396) 
```suggestion
1. [**Installing Carbon for IBM.com**](/developing/web-components-tutorial/step-1)
```